section3/reflection.md

```diff
@@ -2,12 +2,26 @@
 
 1. In Section 3, we removed even more supports and pushed you to learn a bit more independently, read through technical documentation, and use your resources more. How did that feel?
 
+* I'm glad because it forces me to make sure I'm actually learning how to do something instead of just copying down what I need to do. I think it will help me be better.
+
 2. Think of a web or mobile application you use regularly. Where is an example of a place that the developers probably use Arrays? Explain.
 
+* A good example of the use of arrays would be on any social media platform. You will have a friends list and those people who you are friends with will most likely be stored in an array.
+
 3. In your own words, explain what an Array method is.
 
+* An array is a method where you can store multiple items like a list. I think of it as a simple piece of paper you jot down things onto and can reference at any time by looking at the paper. It could be a shopping list, to do list, or a bucket list as an example.
+
 4. What is the purpose of a loop? When we would use a loop in conjunction with array vs. without?
 
+* The purpose of a loop is to repeat the same or similar code a set number of times.
+
+* You would use a loop in conjunction with an array if you want to quickly access all elements in an array. You could use a loop without an array if you wanted to log counting to ten without having an array with 1-10 in it.
+
 5. If you had to teach a 5 year old about loops, how would you explain it?
 
+* A loop in programming is like designing a loop in a roller coaster you set up the length of the ride, how many people are allowed on the ride each time, and how many times you want them to go through a loop on the track. Once the ride is open, the ride will run with these set instructions until it reaches the end.
+
 6. What questions do you still have about Arrays and loops?
+
+* Are we going to cover other types of loops?
```

Comment:
Ohh yeah you will be learning _a lot_ about iteration, and other methods to do it.